Automattic/wp-calypso
# Plans: enable domain to personal plan nudge on domains page

This PR enables the following nudge on the domains page. Clicking on the upgrade button will take you to the checkout page with a Personal plan in cart:

<img width="932" alt="screen shot 2016-10-10 at 3 41 45 pm" src="https://cloud.githubusercontent.com/assets/1270189/19254207/36d88958-8f06-11e6-88a8-ff8875352bb9.png">

## Testing Instructions

- Navigate to calypso.localhost:3000/domains
- Select a wpcom site with: (a free plan + a paid domain )
- Set the abtest value of 'domainToPersonalPlanNudge' to 'nudge'
- The nudge should render
- Clicking on upgrade should take us to checkout
- Clicking the 'x' dismiss should make this item never appear again
